### DISCUSSION TITLE
Fluent + dynamic types.

### DISCUSSION THREAD

**Discussion by mikasoukhov** [linq2db/linq2db#3708] [Q&A] [answered]
Do you have any option create columns by name (string) and type? Without any predefined classes and properties.

Extracted my question from the thread https://github.com/linq2db/linq2db/discussions/3704

**Comment by MaceWindu** [comment, marked as answer]
We don't have such functionality (but you could  always execute raw `CREATE TABLE` sql)

**Comment by mikasoukhov** [comment]
By using ISqlBuilder + SqlTable?

**Reply by MaceWindu** [reply]
Well, in theory it should work (create `SqlCreateTableStatement` and pass it to `ISqlBuilder.BuildSql`).
